nextstrain/cli/runner/aws_batch/__init__.py

```diff
@@ -83,6 +104,8 @@ def run(opts, argv, working_volume = None) -> int:
             name       = run_id,
             queue      = opts.job_queue,
             definition = opts.job_definition,
+            cpus = int(opts.cpus),
+            memory = int(opts.memory),
```

Comment:
Maybe a minor thing, but when code is aligned for readability (on the `=` in this case), I like to keep any newly added code lined up too.

Reply:
I hadn't noticed. Agreed it looks nicer.